### PR TITLE
Support character sets.

### PR DESCRIPTION
**Pull request by mauritsvanrees**
`sr@Latn` and `sr@Cyrl` will be added to language `sr`.
See https://github.com/collective/plone.app.locales/issues/326

Based on the branch of PR #49 for the improved CI, so that one should be merged first.

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -5,6 +5,12 @@
 4.7.1 (unreleased)
```
Comment (icemac) [inline CHANGES.rst:5]: This change should probably increase the minor version as it is a new feature.

**Review by icemac — APPROVED** [review]
LGTM besides one additional wish.

File: CHANGES.rst (hunk 1 above)
Comment (mauritsvanrees) [reply CHANGES.rst]: Done.

**Comment by icemac** [comment]
I just released https://pypi.org/project/zope.i18n/4.8.0/ including these changes.

**Comment by mauritsvanrees** [comment]
Thank you.

**Pull request merged**